railties/test/application/test_runner_test.rb

```diff
@@ -320,6 +320,17 @@ def test_run_relative_path_with_trailing_slash
       end
     end
 
+    def test_run_windows_style_path
+      create_test_file :models, "account"
+      create_test_file :controllers, "accounts_controller"
+
+      # double-escape backslash -- once for Ruby and again for shelling out
+      run_test_command("test\\\\models").tap do |output|
```

Comment:
Windows apparently [ignores double slashes in paths](https://stackoverflow.com/questions/15969608/what-is-the-difference-between-and-in-file-path/15969841#15969841), so this test should hopefully pass on Windows even if the 2nd level of escape is not treated as an escape.

Reply:
> 2nd level of escape is not treated as an escape

One flew over the cuckoo's nest and landed right back in? 😄